### ISSUE TITLE
Dollar math and Sphinx 3.0.1

### ISSUE THREAD

**Issue by jarrodmillman** [matthew-brett/texext#9]
With texext 0.6.5, I am having a problem with dollar math and Sphinx 3.

For instance in networkx, the file [`doc/reference/introduction.rst`](https://github.com/networkx/networkx/blob/master/doc/reference/introduction.rst#data-structure) has
```
As an example, here is a representation of an undirected graph with the
edges $(A, B)$ and $(B, C)$.
```

which in Sphinx 3.0.1 becomes
```
<p>As an example, here is a representation of an undirected graph with the 
edges <span class="math notranslate nohighlight">\(\)</span> and
<span class="math notranslate nohighlight">\(\)</span>.</p>
```
and in Sphinx 2.2.0 becomes
```
<p>As an example, here is a representation of an undirected graph with the
edges <span class="math notranslate nohighlight">\((A, B)\)</span> and
<span class="math notranslate nohighlight">\((B, C)\)</span>.</p>
```

**Comment by matthew-brett** [comment]
Ouch. I will take a look

**Comment by matthew-brett** [comment]
I think fixed with new release 0.6.6 - can you check?

**Comment by jarrodmillman** [comment]
Thanks!  It looks like that worked.
